FirebaseMLModelDownloader/Sources/ModelDownloadConditions.swift

```diff
@@ -15,8 +15,7 @@
 import Foundation
 
 /// Model download conditions.
-// TODO: Implement model download conditions.
 public struct ModelDownloadConditions {
-  // TODO: Intentionally left blank until implementation.
-  public init() {}
+  /// Allow model downloading on a cellular connection. Default is `YES`.
+  public var allowsCellularAccess = true
```

Comment:
Do we have API review for this?

Reply:
We have a review scheduled next week for this. The original approved API didn't have any conditions (or the init).